eng/source-build-patches/0004-Remove-Yarn-dependency-not-used-in-source-build.patch

```diff
@@ -0,0 +1,77 @@
+From 83c0f95c563fa6eb9b446f09166cac2c62ab6390 Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Thu, 14 May 2020 16:29:28 +0000
+Subject: [PATCH 04/14] Remove Yarn dependency - not used in source-build
+
+---
+ eng/targets/Npm.Common.targets                                 | 3 ---
+ global.json                                                    | 1 -
+ .../src/Microsoft.Authentication.WebAssembly.Msal.csproj       | 2 --
+ ...oft.AspNetCore.Components.WebAssembly.Authentication.csproj | 2 --
+ src/Shared/E2ETesting/E2ETesting.targets                       | 3 ---
+ 5 files changed, 11 deletions(-)
+
+diff --git a/eng/targets/Npm.Common.targets b/eng/targets/Npm.Common.targets
+index 696d23402b..6086dd0843 100644
+--- a/eng/targets/Npm.Common.targets
++++ b/eng/targets/Npm.Common.targets
+@@ -1,8 +1,5 @@
+ <Project DefaultTargets="Build" InitialTargets="_CheckForInvalidConfiguration">
+ 
+-  <!-- Version of this SDK is set in global.json -->
+-  <Sdk Name="Yarn.MSBuild" />
+-
+   <PropertyGroup>
+     <NormalizedPackageId>$(PackageId.Replace('@','').Replace('/','-'))</NormalizedPackageId>
+     <PackageFileName>$(NormalizedPackageId)-$(PackageVersion).tgz</PackageFileName>
+diff --git a/global.json b/global.json
+index 501ba161bf..ee9dcaf4fa 100644
+--- a/global.json
++++ b/global.json
+@@ -29,7 +29,6 @@
+     "xcopy-msbuild": "16.5.0-alpha"
+   },
+   "msbuild-sdks": {
+-    "Yarn.MSBuild": "1.22.10",
```

Comment:
As far as I know, global.json and the Arcade SDK don't support conditional SDKs. Is this something the Arcade team is working on❔ Otherwise, what's the recommended approach to handling SDKs we need almost everywhere but in source builds❔

Reply:
I don't have a suggestion.  I'll bring this up in our next source-build sync and will reach out to Arcade on the possibility of supporting this scenario.